src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -261,16 +261,16 @@ export class DBTCoreProjectIntegration
       } else {
         window.showErrorMessage(
           extendErrorWithSupportLinks(
-            "An unexpected error occured while initializing the dbt project: " +
+            "An unexpected error occured while initializing the dbt project at " +
```

Comment:
These are all the Python process closed with exit code errors. The typical python errors we don't show with error message as it might flood the user with error messages.

This error is not particular useful for the user.